locale/en_US/manager.xml

```diff
@@ -502,7 +502,9 @@
 	<message key="manager.navigationMenus.form.pathRegEx">The path field must contain only alphanumeric characters plus '.', '/', '-', and '_'.</message>
 	<message key="manager.navigationMenus.form.duplicatePath">This path already exists for another navigation menu item.</message>
 	<message key="manager.navigationMenus.form.duplicateTitles">This title already exists for another navigation menu.</message>
+	<message key="manager.navigationMenus.form.titleMessage">Provide a custom navigation menu item title</message>	
```

Comment:
I don't think we need this now. The field should just have the section label "Title" and doesn't need a sub-label.